app/src/server.js

```diff
@@ -29,12 +29,30 @@ const passport = require("passport");
 
 const port = process.env.PORT || 443;
 
+// Read security files.
```

Comment:
please add a comment describing the purpose of each file and why they're duplicated